src/wallet/wallet_rpc_server.cpp

```diff
@@ -3807,7 +3807,7 @@ namespace tools
     std::string old_language;
 
     // check the given seed
-    {
+    if (!req.enable_multisig_experimental) {
```

Comment:
As is, `recovery_key` will not be initialized when restoring from multisig seed, which would cause an error if `seed_offset` is also provided.

Should `seed_offset` be supported when restoring from multisig seed?

If not we should assert that `seed_offset` isn't provided when restoring from multisig seed. Otherwise `recovery_key` needs initialized too.

Reply:
Yes `seed_offset` is incompatible with multisig wallets so that condition should be guarded against.